src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -961,6 +961,36 @@ public override SyntaxTree SyntaxTree
 
         internal override IOperation GetOperationWorker(CSharpSyntaxNode node, GetOperationOptions options, CancellationToken cancellationToken)
         {
+            var bindingRoot = GetBindingRootOrFieldInitializer(node);
+
+            var statementOrRootOperation = GetStatementOrRootOperation(bindingRoot, options, cancellationToken);
+            if (statementOrRootOperation == null)
+            {
+                return null;
+            }
+
+            // we might optimize it later
+            return statementOrRootOperation.DescendantsAndSelf().FirstOrDefault(o => !o.IsImplicit && o.Syntax == node);
+        }
+
+        private CSharpSyntaxNode GetBindingRootOrFieldInitializer(CSharpSyntaxNode node)
+        {
+            var bindingRoot = GetBindingRoot(node);
+
+            // if binding root is field variable declarator, make it initializer
```

Comment:
>// if binding root is field variable declarator, make it initializer [](start = 12, length = 68)

It would be good to provide an explanation why we are doing this.

Reply:
added